integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts

```diff
@@ -780,39 +1115,490 @@ medusaIntegrationTestRunner({
             select: ["id", "region_id", "currency_code", "sales_channel_id"],
           })
 
-          await addToCartWorkflow(appContainer).run({
+          await addToCartWorkflow(appContainer).run({
```

Comment:
sugg: Might be worth testing a case for adjustments and tax lines here if custom items do have them.

Reply:
The first iteration of custom line items doesn't support tax lines and adjustments 👍